bin/convert-argv.js

```diff
@@ -148,9 +152,17 @@ module.exports = function(...args) {
 	}
 
 	function processConfiguredOptions(options) {
-		if (options === null || typeof options !== "object") {
+		var webpackConfigurationValidationErrors = validateSchema(
+			webpackConfigurationSchema,
+			options
+		);
+		if (webpackConfigurationValidationErrors.length) {
+			var error = new WebpackOptionsValidationError(
+				webpackConfigurationValidationErrors
+			);
 			console.error(
-				"Config did not export an object or a function returning an object."
+				error.message,
+				`\nReceived: ${typeof options} : ${JSON.stringify(options, null, 2)}`
```

Comment:
Check the json stringify callback to b correct @fokusferit @ematipico

Reply:
I would add a newline `\n` straight after `:`. But the rest looks good to me